src/test/components/__snapshots__/FilterNavigatorBar.test.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`app/ProfileFilterNavigator renders ProfileFilterNavigator properly 1`]
     <span
       class="filterNavigatorBarItemContent"
     >
-      Full Range
+      Full Range (⁨51ms⁩)
```

Comment:
I'm curious: what hidden character do you have, and why is it needed?

Reply:
On second thoughts, I guess they are the directionality characters from Fluent, but didn't realize you need to replicate them in tests.

Reply:
I believe these are the isolation characters inserted by Fluent automatically